src/main/java/org/schedulesdirect/api/Program.java

```diff
@@ -1388,4 +1445,40 @@ private String toString(Collection<?> collection, int maxLen) {
 		builder.append("]");
 		return builder.toString();
 	}
+	
+	/**
+	 * @return the entityType
+	 */
+	public EntityType getEntityType() {
+		return entityType;
+	}
+	
+	/**
+	 * @param entityType the entityType to set
+	 */
+	public void setEntityType(EntityType entityType) {
+		this.entityType = entityType;
+	}
+	
+	/**
+	 * @return the keywords
+	 */
+	public Map<String, List<String>> getKeywords() {
```

Comment:
There doesn't seem to be anything special about keywords.  Just keyword categories & the keywords in those categories, so I went with a Map.